Dockerfile

```diff
@@ -39,15 +47,16 @@ RUN addgroup --system --gid 993 sydent \
     && mkdir /data \
```

Comment:
I realise this isn't yours, but on the line above, why are we creating a password for the sydent user?
It originally has a disabled password. There's no reason to set a static password ... is there?

Reply:
Seems to have been added in #290; wasn't present in #80. No clues in either PR.

Reply:
I reckon we just remove it here as well. A password that is the same for every Sydent container doesn't seem to afford any security anyway.
I also notice `--disabled-login` is an option rather than `--disabled-password`

Excerpt from `/etc/shadow`:
```
dpass:*:19033:0:99999:7:::
dlogin:!:19033:0:99999:7:::
```

> If the password field contains some string that is not a valid result of crypt(3), for instance ! or *, the user will not be able to use a unix password to log in (but the user may log in the system by other means).
> 
> — `man shadow`

`--disabled-login` sounds like it (`!`) will also prevent SSH login using authorised keys. I guess we're not running an SSH daemon anyway, so it probably doesn't matter which one we use.

Reply:
> A password that is the same for every Sydent container doesn't seem to afford any security anyway.

The password comes from /dev/random so I would expect it to be different every time you run build this container. I still think it's odd though.

Reply:
But if it gets uploaded to Docker Hub or something, everyone can then pull that and even crack the password hash if they're so keen. I'd pull it out — the password isn't even being given to anyone so I can't see how it's useful.

Reply:
Oh I see---good point. Yes, let's excise it